docs/concepts/nodes.md

```diff
@@ -11,42 +11,42 @@ NEAR Protocol runs on a collection of publicly maintained computers or "nodes".
 ## Node types
 
 For simplicities sake, NEAR has two main types of nodes: 
-  - block producing nodes (aka "validator nodes")
-  - non-block producing nodes (aka "regular nodes")
+  - Producing nodes (aka "validator nodes")
+  - Non-producing nodes (aka "regular nodes")
 
-### Non-block producing node ("regular node")
+### Non-producing node ("regular node")
 
 Anyone can run a "regular node" by following one of the methods mentioned in [this guide](/docs/local-setup/running-a-node). 
 
 You may decide to run a node of your own for a few reasons:
 
   - To view, process, and validate transactions on `MainNet`, `TestNet` or `BetaNet` (†)
   - To view, process, and validate transactions on an independent / isolated local NEAR network (sometimes called "LocalNet"). (††)
-  - To join `BetaNet` or `MainNet` as a block producing node, aka "Validator Node" (see "[Running a Validator Node](/docs/validator/staking)")
+  - To join `BetaNet` or `MainNet` as a producing node, aka "Validator Node" (see "[Running a Validator Node](/docs/validator/staking)")
 
   _( † ) `TestNet` is intended to operate as similarly to `MainNet`  as possible with only stable releases, while `BetaNet` follows a weekly release cycle._
 
 _( †† ) `LocalNet` would be the right choice if you prefer to avoid leaking information about your work during the development process since `TestNet` and `BetaNet` are *public* networks. `LocalNet` also gives you total control over accounts, economics, and other factors for more advanced use cases (i.e. making changes to `nearcore`)._
 
 
-### Block producing node ("validator node")
+### Producing node ("validator node")
```

Comment:
Why? Producing node makes no sense

Reply:
This was discussed in a thread on slack. @SkidanovAlex was concerned that calling a node "block producing" would be incorrect in the future when chunk producing begins. Do you have a suggestion?

Reply:
No but let's not call them "producing node". For now block producing node is fine